orderer/consensus/hlmirbft/chain.go

```diff
@@ -1041,17 +1141,21 @@ func (c *Chain ) CreateBlock(envs []*common.Envelope) *common.Block {
 	return bc.createNextBlock(envs)
 }
 
+
 //JIRA FLY2-66 proposed changes:Implemented the Snap Function
 func (c *Chain) Snap(networkConfig *msgs.NetworkState_Config, clientsState []*msgs.NetworkState_Client) ([]byte, []*msgs.Reconfiguration, error) {
-
+	var reconfigList []*msgs.Reconfiguration
 	pr := c.Node.PendingReconfigurations
+	for _,value := range pr {
```

Comment:
There's no need to create a separate list to unwrap the`PendingReconfigurations` - pop the first entry and return it.

Reply:
@harrymknight  resolved this change